### ISSUE TITLE
bot.on(ready) is being called more than once

### ISSUE THREAD

**Issue by baabaablackgoat** [MGF1998/SolsticeBot#56]
...not sure if this is intended behaviour, but I don't think it is.

**Comment by baabaablackgoat** [comment]
#57 No major changes, mainly hidden stuff-ish.
Had to do with my settings and my habit of using `ctrl+c` instead of properly logging out via `pls die`

The bot will run ready() again if he loses internet access and reconnects or has to soft-restart from internal errors.
The other situation where it happens is if Discord thinks multiple clients are still "logged in" and they're all on one machine

tldr use the command you dingus
